src/app/services/utils/FormatDate.ts

```diff
@@ -0,0 +1,8 @@
+export const FormatDate = (date: Date): string => {
```

Comment:
You've added `date-fns` library to the project. You can find something useful for date formatting there.